lib/jenkins_pipeline_builder/extension_set.rb

```diff
@@ -65,9 +66,8 @@ def extension
 
       extension = versions[highest_allowed_version]
 
-      unless extension
-        raise "Can't find version of #{name} lte #{installed_version}, available versions: #{versions.keys.map(&:to_s)}"
-      end
+      raise %(Can't find version of #{name} lte #{installed_version}.
+        Available versions: #{versions.keys.map(&:to_s)}) unless extension
```

Comment:
Same, I think this was a rubocop fix that slipped through. Put it back to the way it was.